client/signup/steps/theme-selection/index.jsx

```diff
@@ -126,9 +123,9 @@ const ThemeSelectionStep = React.createClass( {
 				</div>
 				<div className={ themesWrapperClassName } >
 					<StepWrapper
-						fallbackHeaderText={ this.translate( 'Choose a theme.' ) }
-						fallbackSubHeaderText={ this.translate( 'No need to overthink it. You can always switch to a different theme later.' ) }
-						subHeaderText={ this.translate( 'Choose a theme. You can always switch to a different theme later.' ) }
+						fallbackHeaderText={ translate( 'Choose a theme.' ) }
+						fallbackSubHeaderText={ translate( 'No need to overthink it. You can always switch to a different theme later.' ) }
```

Comment:
This line is generating a lint warning, since it exceeds the maximum line length of 140 characters. 
I know that you didn't introduce this but it would be nice to fix that while we are here.